src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -271,15 +289,15 @@ public Event messageID(MessageID attachedMessageID) {
                     .append(description);
         }
         var eventBuilder = com.exactpro.th2.common.grpc.Event.newBuilder()
-                .setId(toEventID(id))
+                .setId(toEventID(bookName, id))
```

Comment:
Let's use the default if null method from the StringUtil class

Reply:
Discussed in chat